bugbug/repository.py

```diff
@@ -653,10 +659,16 @@ def hg_log_multi(repo_dir, revs):
     os.chdir(repo_dir)
 
     CHUNK_SIZE = 256
-    revs_groups = [revs[i : (i + CHUNK_SIZE)] for i in range(0, len(revs), CHUNK_SIZE)]
+    processes = os.cpu_count() + 1
+    CHUNK_SIZE = int(math.ceil(len(revs) / processes))
+    REVS_COUNT = len(revs)
+    revs_groups = [
+        (revs[i], revs[min(i + CHUNK_SIZE, REVS_COUNT) - 1])
+        for i in range(0, REVS_COUNT, CHUNK_SIZE)
+    ]
```

Comment:
Is this change in particular improving performance?

Reply:
Yeah, it isn't crazy but it has an impact

Reply:
I'm asking because we have the same code in `microannotate` :)